### PR TITLE
Stream Cirrus CI webhook events to DataDog

### LINKED ISSUE #1: Cirrus CI audit events + DataDog intergration example

### PR DESCRIPTION
**Pull request by edigaryev**
The resulting event looks in DataDog like this:

Screenshot 2023-07-11 at 22 32 31

Resolves #1.

### CODE REVIEW

File: internal/command/datadog/datadog.go [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+		logger.Warnf("%v", err)
+
+		return ctx.NoContent(http.StatusBadRequest)
+	}
+
+	// Log this event into the DataDog
+	evt := &statsd.Event{
+		Title: fmt.Sprintf("Webhook event of type %s", eventType),
+		Text:  string(body),
+	}
```
Comment (fkorotkov) [inline internal/command/datadog/datadog.go]: Does it support some tags? To annotate it with repository name, user id, etc. Events should be useful and not just some raw dump of webhooks.

Comment (edigaryev) [reply internal/command/datadog/datadog.go]: It seems that DataDog actually treats the `Text` as JSON and allows filtering by it's fields:

Screenshot 2023-07-12 at 00 02 08

I've added some tags in https://github.com/cirruslabs/cirrus-webhooks-server/pull/2/commits/648ecafdf07cf39807146b4f9c8d6e51158532b3 anyway.

File: .cirrus.yml [hunk 2]
```diff
@@ -0,0 +1,34 @@
+task:
+  name: Release Binaries
+  only_if: $CIRRUS_TAG != ''
+  depends_on:
+    - Lint
+    - Test (macOS)
```
Comment (fkorotkov) [inline .cirrus.yml]: Don't think there is such task.

File: .cirrus.yml [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+    memory: 12G
+  release_script: goreleaser
+
+
+docker_builder:
+  name: Release Docker Image
+  only_if: $CIRRUS_TAG != ''
+  depends_on:
+    - Lint
+    - Test (macOS)
```
Comment (fkorotkov) [inline .cirrus.yml]: Same here.

File: README.md [hunk 4]
```diff
@@ -1,2 +1,43 @@
-# cirrus-webhooks-server-example
-Example of a simple server that processes webhook events from Cirrus CI
+# Cirrus Webhooks Server
+
+Examples of the webhook event processors from the Cirrus CI.
+
+## Processors
+
+## DataDog
```
Comment (fkorotkov) [inline README.md]: This whole section looks like it belong to something like `Dvelopment.md`. Readme is usually for setup steps. Like here is not to get a Datadog API and what's the environment variable that the server expects. Here is now to run the server on the default port and that's how we can we specify the port. Could you please take a look at how this section can be more user friendly and not oriented on developing the server itself?

File: .cirrus.yml (hunk 3 above)
Comment (edigaryev) [reply .cirrus.yml]: Removed in https://github.com/cirruslabs/cirrus-webhooks-server/pull/2/commits/dab7a1a0ed18ff4a22091f83dd1af5bd4fbb6d53.

File: .cirrus.yml (hunk 2 above)
Comment (edigaryev) [reply .cirrus.yml]: Removed in https://github.com/cirruslabs/cirrus-webhooks-server/pull/2/commits/dab7a1a0ed18ff4a22091f83dd1af5bd4fbb6d53.

File: README.md (hunk 4 above)
Comment (edigaryev) [reply README.md]: >Like here is not to get a Datadog API and what's the environment variable that the server expects.

Added in https://github.com/cirruslabs/cirrus-webhooks-server/pull/2/commits/deb70c898f5576a5a49553668133948d8825fab9.

**Review by fkorotkov — APPROVED** [review]

**Pull request merged**